cs/test/expressions/Clone.expressions

```diff
@@ -809,6 +809,46 @@
                 } .Else {
                     .Default(System.Void)
                 };
+                .Block(System.Object $obj) {
+                    $obj = (System.Object)$Example_obj._dt;
+                    .Block() {
+                        .Block(ExpressionsTest.Generic`1[System.DateTime] $Generic`1_obj) {
+                            $Generic`1_obj = (ExpressionsTest.Generic`1[System.DateTime])$obj;
+                            .Default(System.Void);
+                            .If ($Generic`1_obj.value != .Call ExpressionsTest.BondTypeAliasConverter.Convert(
+                                0L,
+                                .Default(System.DateTime))) {
+                                ($Example_result._dt).value = .Call ExpressionsTest.BondTypeAliasConverter.Convert(
+                                    .Call ExpressionsTest.BondTypeAliasConverter.Convert(
```

Comment:
I'm trying to decipher if we are calling convert() unnecesary too many times. We do one when comparing on if with a dummy value just to get the type check, then it looks like we call twice here? looks like a nested convert but it does not make sense. I'm not sure, thoughts?

Reply:
This is unrelated to this change. Same can be seen at [448-452](https://github.com/ara-ayvazyan/bond/blob/2fb352f12815603d735ec693b374310768c5c96b/cs/test/expressions/Clone.expressions#L448-L452).

Reply:
True.

Reply:
Actually, I'm not sure which value must be compared with: `default(int64)` or `default(DateTime)`, since the user-defined conversion may not map these values.